availability-store/Cargo.toml

```diff
@@ -15,15 +15,15 @@ futures = "0.3.1"
 tokio = { version = "0.2.4", features = ["rt-core"] }
 exit-future = "0.2.0"
 codec = { package = "parity-scale-codec", version = "1.1.0", features = ["derive"] }
-sc-network = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
```

Comment:
can you point this back to `polkadot-master`?

Reply:
Just this one dep?

Reply:
no, all of them...